### PR TITLE
Fix disabled tests in MultipleLoopTests.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @halter73 @mikeharder

### CODE REVIEW

File: test/Microsoft.AspNetCore.Server.KestrelTests/MultipleLoopTests.cs [hunk 1]
```diff
@@ -97,16 +101,16 @@ public void ServerPipeListenForConnections()
...
                     connect.Dispose();
+
                     clientConnectionPipe.ReadStart(
-                        (_3, cb, _4) => buf,
-                        (_3, status2, _4) =>
+                        (handle2, cb, state2) => buf,
+                        (handle2, status2, state2) =>
                         {
-                            if (status2 == 0)
+                            if (status2 == Constants.EOF)
```
Comment (cesarbs) [inline test/Microsoft.AspNetCore.Server.KestrelTests/MultipleLoopTests.cs]: This is the actual fix.

**Comment by mikeharder** [comment]
:shipit:

**Comment by halter73** [comment]
:shipit:

**Comment by halter73** [comment]
Thanks for looking into this!

**Pull request merged**